src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,41 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        int totalSupply = 0;
+        int totalBuy = 0;
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            String line;
+            while ((line = reader.readLine()) != null) {
+                String[] record = line.split(",");
+                String operationType = record[0];
+                int amount = Integer.parseInt(record[1]);
+                if ("supply".equals(operationType)) {
+                    totalSupply += amount;
+                } else if ("buy".equals(operationType)) {
+                    totalBuy += amount;
+                }
+            }
```

Comment:
Consider using constants for the strings 'supply' and 'buy' to avoid magic strings in the code. This will make it easier to manage and change these values in the future if needed.